views/partials/forms/scan-email-form.hbs

```diff
@@ -15,7 +24,7 @@
     {{#if featuredBreach}}
       <input type="submit" value="{{ getString "find-out" }}"/>
     {{else}}
-      <input type="submit" value="{{ getString "check-for-breaches" }}"/>
+      <input {{#if isUserInExperiment}} {{> analytics/fxa id="fx-monitor-check-for-breaches-blue-btn" }} {{/if}} type="submit" value="{{ getString "check-for-breaches" }}"/>
```

Comment:
It looks like the FxA `metrics-flow` values are not being added to this element, so we need to just add something like `class="add-metrics-flow-values"` after the `{{#if isUserInExperiment}}` and then add that class to the selector on L156 of `fxa-analytics.js` so that it looks  something like this:

```
document.querySelectorAll(".open-oauth, .add-metrics-flow-values").forEach( async(el) => {
```